packages/relay/src/lib/eth.ts

```diff
@@ -1014,6 +1014,13 @@ export class EthImpl implements Eth {
    * @param blockNumber
    */
   async getCode(address: string, blockNumber: string | null, requestIdPrefix?: string) {
+    if (!EthImpl.isBlockParamValid(blockNumber)) {
+      throw predefined.INVALID_PARAMETER(
```

Comment:
Can use predefined.UNKNOWN_BLOCK.
Feel free to update with message

Reply:
Right UNKNOWN_BLOCK would make more sense in this case! Thanks for pointing it out!